### PR TITLE
chore(suite): hide polygon trade in trade section

### PR DESCRIPTION
**Pull request by tomasklim**
## Description

- hide polygon missing MATIC banner when not in debug mode
- hide add polygon account when not in debug mode

## Screenshots:

**debug mode**
Screenshot 2024-02-07 at 14 49 13
Screenshot 2024-02-07 at 14 48 58

**no debug mode** 
Screenshot 2024-02-07 at 14 49 38
Screenshot 2024-02-07 at 14 49 26

### CODE REVIEW

File: packages/suite/src/hooks/wallet/useCoinmarketExchangeOffers.ts [hunk 1]
```diff
@@ -61,6 +61,7 @@ export const useOffers = ({ selectedAccount }: UseCoinmarketExchangeFormProps) =
     const device = useSelector(selectDevice);
     const { addressVerified, dexQuotes, exchangeInfo, fixedQuotes, floatQuotes, quotesRequest } =
         useSelector(state => state.wallet.coinmarket.exchange);
+    const isDebug = useSelector(state => state.suite.settings.debug.showDebugMenu);
```
Comment (komret) [inline packages/suite/src/hooks/wallet/useCoinmarketExchangeOffers.ts]:
```suggestion
    const isDebug = useSelector(selectIsDebugModeActive);
```

File: packages/suite/src/views/wallet/coinmarket/exchange/components/ExchangeForm/Inputs/index.tsx [hunk 2]
```diff
@@ -63,6 +63,7 @@ const Inputs = () => {
     const { shouldSendInSats } = useBitcoinAmountUnit(account.symbol);
     const deviceAccounts = useSelector(selectDeviceAccounts);
     const { elevation } = useElevation();
+    const isDebug = useSelector(state => state.suite.settings.debug.showDebugMenu);
```
Comment (komret) [inline packages/suite/src/views/wallet/coinmarket/exchange/components/ExchangeForm/Inputs/index.tsx]:
```suggestion
    const isDebug = useSelector(selectIsDebugModeActive);
```

**Review by komret — CHANGES_REQUESTED** [review]

File: packages/suite/src/views/wallet/coinmarket/exchange/components/ExchangeForm/Inputs/index.tsx (hunk 2 above)
Comment (tomasklim) [reply packages/suite/src/views/wallet/coinmarket/exchange/components/ExchangeForm/Inputs/index.tsx]: f131005898b5bc0d6fd743150cfc7ed0552f25e6

File: packages/suite/src/hooks/wallet/useCoinmarketExchangeOffers.ts (hunk 1 above)
Comment (tomasklim) [reply packages/suite/src/hooks/wallet/useCoinmarketExchangeOffers.ts]: f131005898b5bc0d6fd743150cfc7ed0552f25e6

**Review by komret — APPROVED** [review]

**Pull request merged**